src/create-module-declaration.js

```diff
@@ -391,24 +398,22 @@ export function create_module_declaration(id, entry, created, resolve) {
 			if (mod) content += '\n' + mod;
 		}
 
-		if (export_specifiers.length > 0) {
-			content += `\n\n\texport { ${export_specifiers.join(', ')} };`;
-		}
-
 		// finally, export any bindings that are exported from external modules
 
-		/** @type {string[]} */
-		const specifiers = [];
-
 		for (const name of exports) {
 			const declaration = trace_export(entry, name);
 			if (declaration?.external) {
-				specifiers.push(declaration.alias);
+				export_specifiers.push(declaration.alias);
 			}
 		}
 
-		if (specifiers.length > 0) {
```

Comment:
would it be simpler to just get rid of this condition, so that the `export` is always generated, but otherwise keep everything the same?

Reply:
ignore me